internal/controllers/clusterextension_controller.go

```diff
@@ -574,7 +572,6 @@ func (r *ClusterExtensionReconciler) SetupWithManager(mgr ctrl.Manager) error {
 				return true
 			},
 		}).
-		Watches(&corev1.Pod{}, mapOwneeToOwnerHandler(mgr.GetClient(), mgr.GetLogger(), &ocv1alpha1.ClusterExtension{})).
```

Comment:
Since we're on it - we should also be able to remove the rbac too, to watch pods: https://github.com/operator-framework/operator-controller/blob/cb63023e916dc302548ce841c7e5540303ad2f2b/internal/controllers/clusterextension_controller.go#L105-L107.

If not in this PR, it can be done in follow up too

Reply:
I can do it now.

Reply:
Pushed a change. I removed the pods stuff, and changed the configmaps RBAC (what was that for?) to secrets since the helm stuff needs to manage release secrets.

It's all sorta moot though because:
- We give ourselves `*/*/*` in the next line.
- We'll eventually drop the secrets permission here because we'll expect the SA to have the necessary permission to handle their own release secret.

Reply:
> and changed the configmaps RBAC (what was that for?)

I _think_ it may have gotten ported over from Extension, where Kapp needed CRUD permissions for configmaps.